index_twitter_stream.py

```diff
@@ -18,8 +19,8 @@
 # Get elasticsearch connection
 es = esconn.esconn()
 
-if len(sys.argv) >= 2:
-    sys.exit('ERROR: Received 2 or more arguments. Expected 1: Topic file name')
+if len(sys.argv) > 2:
```

Comment:
Since exactly two arguments is valid checking for `>=2` would improperly catch valid cases